app/components/ChartTypeSelector/index.js

```diff
@@ -60,6 +62,7 @@ class ChartTypeSelector extends Component {
       disabled,
       checked: (typeConfig.type === this._getChartType()),
       onChange: (evt) => this._selectChartType(evt.target.value),
+      style: disabled ? { cursor: 'default' } : {},
```

Comment:
You're already passing in `disabled` into the radio component, I think this style change should be in that component. Keep style modifications localized as possible.